GoogleForCreators/web-stories-wp
# Fix zooming for media by displaying overflow in edit mode

## Summary

Fixes zooming from media, specifically in case of background media and when the media gets "over the limits" of a Page while in edit mode.


## Relevant Technical Choices

Stops hiding overflow for edit layer if it uses the grayoutMode. Using `grayoutMode` also makes the overflow relevant by adding the parts of media that are over the Page limit due to zooming and also the zoom slider itself.
## To-do



## User-facing changes
- In media edit mode, overflow is now visible.


## Testing Instructions

I
1. Add media.
2. Set it as background
3. Go to edit mode
4. Verify that zooming slider is visible and functional.

II
1. Add media
2. Resize it to half the page and move it to the bottom of the Page
3. Go to edit mode
4. Verify that the zooming slider is visible.
5. Zoom the media so that its edges would go over the Page
6. Verify that the image is not cut off from the area that goes over the Page limits.



---



Fixes #1491
